helpers/labgrid-raw-interface

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+#
+# Wrapper script to be deployed on machines whose network interfaces should be
+# controllable via the RawNetworkInterfaceDriver. A /etc/labgrid/helpers.yaml
+# can deny access to network interfaces. See below.
+#
+# This is intended to be used via sudo. For example, add via visudo:
+# %developers ALL = NOPASSWD: /usr/local/sbin/labgrid-raw-interface
+
+import argparse
+import os
+import sys
+import warnings
+
+import yaml
+
+
+def get_denylist():
+    try:
+        with open("/etc/labgrid/helpers.yaml") as stream:
+            data = yaml.load(stream, Loader=yaml.SafeLoader)
+    except (PermissionError, FileNotFoundError, AttributeError):
+        warnings.warn("No configuration file (/etc/labgrid/helpers.yaml), inaccessable or invalid yaml, continuing without denylist.")
+        return []
+
+    denylist = data.get("raw-interface", {}).get("denied-interfaces", [])
+
+    if not isinstance(denylist, list):
+        warnings.warn("No explicit denied-interfaces or not a list, please check your configuration.")
+
+    denylist.append("lo")
```

Comment:
If `denylist` is not a list, `.append("lo")` will fail. We should just abort here.

Reply:
Done.